test/test_auth.py

```diff
@@ -406,6 +413,86 @@ def test_auth_query_database_setting(
                 )
 
 
+@pytest.mark.skipif(
+    "FREEBSD",
+    reason="FreeBSD does not seem to observe the expected OperationalError description",
```

Comment:
What error does it give instead?

Reply:
The cirrus test says "psycopg.OperationalError: consuming input failed: server closed the connection unexpectedly". The bouncer log does seem to correctly show the error though... you know what, let me rework the test to look at the bouncer log for confirmation the right error is happening.

Reply:
...and it looks like the new test works well.